pandas/tests/io/json/test_readlines.py

```diff
@@ -13,49 +14,64 @@
 
 from pandas.io.json._json import JsonReader
 
+xfail_json_engine_pyarrow = pytest.mark.usefixtures("json_engine_pyarrow_xfail")
+
 
 @pytest.fixture
 def lines_json_df():
     df = DataFrame({"A": [1, 2, 3], "B": [4, 5, 6]})
     return df.to_json(lines=True, orient="records")
 
 
-def test_read_jsonl():
+@xfail_json_engine_pyarrow
+def test_read_jsonl(engine):
     # GH9180
-    result = read_json('{"a": 1, "b": 2}\n{"b":2, "a" :1}\n', lines=True)
+    result = read_json('{"a": 1, "b": 2}\n{"b":2, "a" :1}\n', lines=True, engine=engine)
     expected = DataFrame([[1, 2], [1, 2]], columns=["a", "b"])
     tm.assert_frame_equal(result, expected)
 
 
-def test_read_datetime():
+def test_read_jsonl_engine_pyarrow(json_dir_path, engine):
+    result = read_json(
+        os.path.join(json_dir_path, "line_delimited.json"),
+        lines=True,
+        engine=engine,
+    )
+    expected = DataFrame({"a": [1, 3, 5], "b": [2, 4, 6]})
+    tm.assert_frame_equal(result, expected)
+
+
+@xfail_json_engine_pyarrow
```

Comment:
Can we supply the xfail with a comment? Might be helpful to know why each case is getting skipped